planner_cspace/test/src/test_debug_outputs.cpp

```diff
@@ -49,6 +51,7 @@ class DebugOutputsTest : public ::testing::Test
     // Wait planner
     while (ros::ok())
     {
+      publishTransform(2.5, 0.5, 3.14);
```

Comment:
Is it related to this PR?

Reply:
(`time` - `last_time`) will not be true because `start_` will not be updated without this process.
The values are borrowed from `planner_cspace/test/test/debug_outputs_rostest.test`.

But because I'll remove the check of timestamp (as you mentioned above), this `publishTransform()` will not be needed.

Reply:
Should I restore this test to the previous implementation?

Reply:
It seems better to revert (or separate PR for this change if needed)